src/ClusterBootstrap/services/jobmanager/jobmanager.yaml

```diff
@@ -80,6 +80,14 @@ spec:
           tcpSocket:
             port: 9200
           timeoutSeconds: 10
+      - name: redis
```

Comment:
python redis client behaviour is it will not throw any exception on connection. Will throw exception on actions. I have caught the exception on action. It will ignore the exception, and the process will just do not emit any job metrics. I think this behaviour is reasonable.